certbot-apache/certbot_apache/interfaces.py

```diff
@@ -198,6 +199,11 @@ def find_ancestors(self, name):
         Traverses the ancestor tree up, searching for BlockNodes with a specific
         name.
 
+        .. note:: To maintain consistency with the original parser, this search
+                  function currently only traverses nodes in the same file. Due
+                  to this, we also don't run into the multiple-includes
+                  ambiguity that the `ancestors` property has, for the time being.
```

Comment:
Just a note: This is going to change when #7642 (and a subsequent PR leveraging it for ParserNode interfaces) lands.